android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -504,6 +504,14 @@ private PackageInfo getPackageInfo() throws Exception {
     return getReactApplicationContext().getPackageManager().getPackageInfo(getReactApplicationContext().getPackageName(), 0);
   }
 
+  private String getInstallerPackageNameSync() {
+    String packageName = getReactApplicationContext().getPackageName();
+    return getReactApplicationContext().getPackageManager().getInstallerPackageName(packageName);
```

Comment:
getInstallerPackageName is [documented to possibly return null](https://developer.android.com/reference/android/content/pm/PackageManager#getInstallerPackageName(java.lang.String)), instead of having a tri-state return value (something useful, "unknown", or null) I think it's a better dev experience if it's only 2 states (something useful or unknown). 

```suggestion
    String installerPackageName = getReactApplicationContext().getPackageManager().getInstallerPackageName(packageName);
    if (installerPackageName == null) {
      return "unknown";
    }
    return installerPackageName;
```

An alternative would be to check for null return in the javascript and mutate it to unknown there, but I think handling it in Java is fine. I just coded that in the comment text box so triple check it, if you like it, before committing :-)